darts/metrics/metrics.py

```diff
@@ -941,3 +960,100 @@ def dtw_metric(actual_series: Union[TimeSeries, Sequence[TimeSeries]],
+    (containing N samples) :math:`\\hat{y}_t` of shape :math:`T \\times N`, rho-risk is a metric that quantifies the
+    accuracy of a specific quantile :math:`\\rho` from the predicted value distribution.
+
+    For a univariate stochastic predicted TimeSeries the :math:`\\rho`-risk is given by:
+
+    .. math:: \\frac{ L_{\\rho} \\left( Z, \\hat{Z}_{\\rho} \\right) } {Z},
+
+    where :math:`L_{\\rho} \\left( Z, \\hat{Z}_{\\rho} \\right)` is`the :math:`\\rho`-loss function:
+
+    .. math:: L_{\\rho} \\left( Z, \\hat{Z}_{\\rho} \\right) = 2 \\left( \\hat{Z}_{\\rho} - Z \\right)
+        \\left( \\rho I_{\\hat{Z}_{\\rho} > Z} - \\left( 1 - \\rho \\right) I_{\\hat{Z}_{\\rho} \leq Z} \\right),
+
+    where :math:`Z = \\sum_{t=1}^{T} y_t` (1) is the aggregated target value and :math:`\\hat{Z}_{\\rho}` is the
+    :math:`\\rho`-quantile of the predicted values. For this, each sample realization :math:`i \\in N` is first aggregated over the
+    time span similar to (1) with :math:`\\hat{Z}_{i} = \\sum_{t=1}^{T} \\hat{y}_{i,t}`.
+
+    :math:`I_{cond} = 1` if cond is True else :math:`0``
+
+    Parameters
+    ----------
+    actual_series
+        The `TimeSeries` or `Sequence[TimeSeries]` of actual values.
+    pred_series
+        The `TimeSeries` or `Sequence[TimeSeries]` of predicted values.
+    rho
+        The quantile (float [0, 1]) of interest for the risk evaluation.
+    intersect
+        For time series that are overlapping in time without having the same time index, setting `intersect=True`
+        will consider the values only over their common time interval (intersection in time).
+    reduction
+        Function taking as input a `np.ndarray` and returning a scalar value. This function is used to aggregate
+        the metrics of different components in case of multivariate `TimeSeries` instances.
+    inter_reduction
+        Function taking as input a `np.ndarray` and returning either a scalar value or a `np.ndarray`.
+        This function can be used to aggregate the metrics of different series in case the metric is evaluated on a
+        `Sequence[TimeSeries]`. Defaults to the identity function, which returns the pairwise metrics for each pair
+        of `TimeSeries` received in input. Example: `inter_reduction=np.mean`, will return the average of the pairwise
+        metrics.
+    n_jobs
+        The number of jobs to run in parallel. Parallel jobs are created only when a `Sequence[TimeSeries]` is
+        passed as input, parallelising operations regarding different `TimeSeries`. Defaults to `1`
+        (sequential). Setting the parameter to `-1` means using all the available processors.
+    verbose
+        Optionally, whether to print operations progress
+
+    Returns
+    -------
+    float
+        The rho-risk metric
+    """
+
+    def rho_loss(actual_series: Union[TimeSeries, Sequence[TimeSeries]],
```

Comment:
Any specific reason for having a function within a function here?

Reply:
Yeah, I thought the same. It was meant for me to make it more clear.
But since it doesn't have any functional reason, I will remove it.